src/modifier/modifier.ts

```diff
@@ -506,6 +506,7 @@ export abstract class PokemonHeldItemModifier extends PersistentModifier {
       if (pokemon) {
         const pokemonIcon = scene.addPokemonIcon(pokemon, -2, 10, 0, 0.5);
         container.add(pokemonIcon);
+        container.setName((pokemon.id).toString());
```

Comment:
```suggestion
        container.setName(pokemon.id.toString());
```
NIT: Unnecessary parentheses I think